gobblin-utility/src/main/java/org/apache/gobblin/util/retry/RetryerFactory.java

```diff
@@ -53,10 +59,14 @@ public class RetryerFactory<T> {
   // value large or equal to 1
   public static final String RETRY_TIMES = "retry_times";
 
-  private static final Predicate<Throwable> RETRY_EXCEPTION_PREDICATE;
+  // Key to store a comma-separated list of exception class names that should be retried
+  public static final String EXCEPTION_LIST_FOR_RETRY_CONFIG_KEY = "EXCEPTION_LIST_FOR_RETRY";
```

Comment:
maybe we can move this line along with https://github.com/apache/gobblin/blob/master/gobblin-api/src/main/java/org/apache/gobblin/configuration/ConfigurationKeys.java#L1063

Reply:
makes sense, addressed